src/components/DetailsList.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/prop-types */
+import style from './components.module.css';
+
+function DetailsList({ country }) {
+  return (
+    <div className="detail-container">
+      <div className="detail-flag">
+        <img src={country.flags.svg} alt={country.name.common} className={style.image} />
+        <p>{country.name.official}</p>
+      </div>
+      <div className="space-global">
+        <div className="space-elements">
+          <p>Common name :</p>
+          <p>{country.name.common}</p>
+        </div>
+        <div className="space-elements">
+          <p>Capital :</p>
+          <p>{country.capital[0]}</p>
+        </div>
+        <div className="space-elements">
+          <p>Population :</p>
+          <p>{country.population}</p>
+        </div>
+        <div className="space-elements">
+          <p>
+            Languages:
+            {' '}
+            {' '}
+          </p>
+          <p>
+            {Object.values(country.languages).map((value) => (
+              <span key={value}>
+                {'  '}
+                {value}
+                {' ,  '}
+              </span>
+            ))}
+
+          </p>
+        </div>
+        <div className="space-elements">
+          <p>Currencies:</p>
+          <p>
+            {Object.keys(country.currencies).map((value) => (
+              <span key={value}>
+                {'   '}
+                {value}
+                {' '}
+              </span>
+            ))}
+
+          </p>
+        </div>
+        <div className="space-elements">
+          <p>Continent :</p>
+          <p>{country.continents}</p>
+        </div>
+        <div className="space-elements">
+          <p>Subregion :</p>
+          {'   '}
+          <p>{country.subregion}</p>
+        </div>
+        <div className="space-elements">
+          <p>Time Zone:</p>
+          <p>{country.timezones[0]}</p>
+        </div>
+        <div className="space-elements">
+          <p>Start of the week:</p>
+          <p>{country.startOfWeek}</p>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default DetailsList;
```

Comment:
@Ismailco this is the updated files, please kindly re-review my project i have implemented the requested changes from the previous reviewer

Reply:
@Pamphilemkp, Yes, you fixed some linters issues but there are still a bunch of `eslint-disable` in the other files and `/* eslint-disable react/prop-types */` in this file. kindly fix them so you don't need to use `eslint-disable` at all

Reply:
I have updated the UI but it would be better if I have all the subject changes at once, so that I Know what to do instead after implementing them then other changes come.
please kindly subjects all the changes i have to make at once.